### ISSUE TITLE
Generate badges for each framework

### ISSUE THREAD

**Issue by jsbean** [dn-m/dn-m.github.io#33]
Create script that generates badges of various types, and inserts them into the relavent `README.md` file.

``` Markdown
[![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/Carthage/Carthage) [![Build Status](https://travis-ci.org/dn-m/$FRAMEWORKNAME.svg?branch=master)](https://travis-ci.org/dn-m/$FRAMEWORKNAME)
```

**Comment by delucis** [comment]
Can this be combined with #32? Looks like the badges go just above the block you want to generate there.

**Comment by jsbean** [comment]
Yes. I'll combine them.

**Comment by jsbean** [comment]
Included in #34.

**Comment by jsbean** [comment]
Migrated to [DocumentationTools](https://github.com/dn-m/DocumentationTools/issues).
